src/components/Button/Style.js

```diff
@@ -2,9 +2,16 @@ import { Button } from '@material-ui/core';
 import styled from 'styled-components';
 
 const StyledButton = styled(Button)`
+  font-size: 20px;
+  letter-spacing: 2px;
+  margin-bottom: 10px;
+  max-width: ${({ maxwidth }) => maxwidth};
+  min-width: ${({ minwidth }) => minwidth};
+  background-color: #e91e63;
+  color: white;
   ${({ theme }) => `
+  
    &:hover {
-        background-color: ${theme.palette.primary.main};
```

Comment:
Why do we need the :hover suedo class styling specifier when the styling has been remove .. please remove the specifier above..